docs/source/conf.py

```diff
@@ -10,10 +10,13 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
-# import os
```

Comment:
Removing that line makes the example incomplete. In my opinion the example should either be removed entirely or left as-is.